.npmignore

```diff
@@ -1,2 +1,5 @@
 test/*
-
+.editorconfig
+.eslintrc
```

Comment:
`npm explore foo && npm install && npm test` will work just fine, and calls through to `npm run lint`.